apache/kafka
# KAFKA-15471 [MINOR]: Fix backward-compatibility bug

*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

## Manually tested to ensure backward compatibility
- Start a controller process with` haileyni@MDQ0VWMW22 kraft % ../../bin/kafka-server-start.sh controller.properties`
- Kill with `haileyni@MDQ0VWMW22 kafka % bin/kafka-server-stop.sh`
- The controller process is killed, which is the same behavior as before the KIP (all kafka processes are killed when no flags are provided), so we ensure backward compatibility.

### Committer Checklist (excluded from commit message)
